tests/unit/lib/foxbox/event-dispatcher_test.js

```diff
@@ -0,0 +1,423 @@
+import EventDispatcher from 'js/lib/foxbox/event-dispatcher';
+
+describe('EventDispatcher >', function () {
```

Comment:
Probably not, "historically" I used ```function``` in tests and where I don't need to grab context (likely browser knows that and doesn't do anything especial, but I don't know for sure), so just used to do it.

Do you think we should use arrows here?